applications/odc-monitoring-project-and-application-metrics-using-developer-perspective.adoc

```diff
@@ -34,6 +34,6 @@ include::modules/odc-monitoring-your-app-vulnerabilities.adoc[leveloffset=+1]
 ifdef::openshift-rosa,openshift-dedicated[]
```

Comment:
You may want to move the conditionals up a line since HCP gets an empty Additional Resources

![image](https://github.com/user-attachments/assets/abe1d247-ea6e-455e-abb6-ceea2f5d738e)

Reply:
Bah! Thank you!